requirements.txt

```diff
@@ -13,3 +13,6 @@ pytest==6.2.5
 pytest-flask==1.2.0
 markdown==3.3.3
 beautifulsoup4==4.10.0
+udata_event_service==0.0.7
+# This can be removed once Flask is updated to 2.1.1
+werkzeug==2.0.3
```

Comment:
Couldn't we update Flask in this PR instead?

Reply:
Yes